README.md

```diff
@@ -342,6 +342,30 @@ While several Ruby books suggest the first style, the second is much more promin
 
 Refactoring is even better. It's worth looking hard at any code that explicitly checks types.
 
+- For single-line lambdas, `lambda` or `->` may be used.
+
+```ruby
+  scope :example, -> { where(:a => b) }
+
+  # Or
+
+  scope :example, lambda { where(:a => b) }
+```
+
+- For multi-line lambdas, prefer `lambda do ... end`
+
+```ruby
+  # bad
+  scope :long_example, lambda {
+    # ...
+  }
+
+  # good
+  scope :long_example, (lambda do
```

Comment:
Actually, looking at the upstream style guide, there's no explicit "use `do … end`" rule — it's just implied by the examples (there's no "bad" version). I'd say we stay equally vague here and assume people will be able to use their judgement when dealing with multi-line `lambdas`. The key parts really are about when and how to prefer stabby lambdas